src/powfacpy/case_studies.py

```diff
@@ -219,29 +227,89 @@ def set_parameters(self, case_obj_or_case_num):
       if value is not None:
         self.set_attr_by_path(path, value)  
 
-  def get_study_cases(self, conditions):
+  def get_study_cases(self, conditions, return_case_numbers=False):
     """Retrieve study case objects depending on parameter values.
     Arguments:
-      conditions: A dictionary with
-        keys: parameter names
-        values: lambda function with boolean return value depending on 
-          parameter (key)
+      conditions: 
+        Either a dictionary with
+          keys: parameter names
+          values: lambda function with boolean return value depending on 
```

Comment:
```suggestion
          values: lambda functions with boolean return value depending on 
```
would it be a lambda function for each key?

Reply:
yes